docs/reference/objects/product/extra.md

```diff
@@ -10,6 +10,14 @@ grand_parent: Objects
 object
 {: .label .fs-1 }
 
+
+The extra object will behave slightly differently depending on how it is accessed.
+- When accessed through an [experience date]({% link docs/reference/objects/product/experience_date.md %}), some methods will return a result that is specific to that given date.
+- When accessed independently or through a product, those methods will take into account all dates on the product.
```

Comment:
Following on from the slack convo just want to clarify, would this also be for just upcoming dates?
```suggestion
- When accessed independently or through a product, those methods will take into account all upcoming dates on the product.
```